tests/framework/utils_cpuid.py

```diff
@@ -101,3 +102,35 @@ def check_guest_cpuid_output(
         "some keys in dictionary have not been found in the output: %s"
         % expected_key_value_store
     )
+
+
+def build_cpuid_dict(raw_cpuid_output):
+    """Build CPUID dict based on raw cpuid output"""
+    cpuid_dict = {}
+    ptrn = re.compile("^ *(.*) (.*): eax=(.*) ebx=(.*) ecx=(.*) edx=(.*)$")
+    for line in raw_cpuid_output:
+        res = ptrn.findall(line)
+        if len(res):
+            leaf, subleaf, eax, ebx, ecx, edx = res[0]
```

Comment:
Maybe 

```python
... = [int(x, 16) for x in res[0]]
```

so you don't have to convert below?

Reply:
done